executor/aggfuncs/func_percentile.go

```diff
@@ -68,6 +68,15 @@ func (e *basePercentile) AppendFinalResult2Chunk(_ sessionctx.Context, _ Partial
 	return nil
 }
 
+// TODO implement it
+func (e *basePercentile) SerializeForSpill(_ sessionctx.Context, partialResult PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper) {
```

Comment:
We can put this empty implementation on baseAggFunc, and the other empty implementations can be removed.

``` go
func (*baseAggFunc) SerializeForSpill(){}
func (*baseAggFunc) DeserializeToPartialResultForSpill(){}
```

Reply:
> We can put this empty implementation on baseAggFunc, and the other empty implementations can be removed.
> 
> ```go
> func (*baseAggFunc) SerializeForSpill(){}
> func (*baseAggFunc) DeserializeToPartialResultForSpill(){}
> ```

okk